core/src/test/java/org/apache/accumulo/core/iterators/user/CombinerTest.java

```diff
@@ -922,4 +923,49 @@ public void testDeleteHandling() throws Exception {
     runDeleteHandlingTest(input, expected, null, paritalMajcIe, ".*ERROR.*SummingCombiner.*ACCUMULO-2232.*");
     runDeleteHandlingTest(input, expected, null, fullMajcIe, ".*ERROR.*SummingCombiner.*ACCUMULO-2232.*");
   }
+
+  /**
+   * Tests the Lossy option will ignore errors in TypedValueCombiner. Uses SummingArrayCombiner to generate error.
+   */
+  @Test
+  public void testLossyOption() throws IOException, IllegalAccessException, InstantiationException {
+    Encoder<List<Long>> encoder = SummingArrayCombiner.VarLongArrayEncoder.class.newInstance();
```

Comment:
Whoops... weird copy and paste hold over. Fixed in 4262a314ce37d822ec96030872f10c90a8b18d8